components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/openidconnect/RequestObjectValidatorImpl.java

```diff
@@ -280,6 +319,27 @@ protected boolean isSignatureVerified(SignedJWT signedJWT, Certificate x509Certi
         }
     }
 
+    /**
+     * Validate the signedJWT signature when jwks end point is given instead of public certificate.
+     *
+     * @param jwtString
+     * @param jwksUri
+     * @param algorithm
+     * @param options
+     * @return
+     */
+    protected boolean isSignatureVerified(String jwtString, String jwksUri, String algorithm, Map<String, Object> options) throws OAuthClientAuthnException {
```

Comment:
Doesn't this line surpass the allowed limit of 120 ?

Reply:
resolved